CocktailList.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from "react";
+import { useRef, useState } from "react";
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchSearchCocktail } from "../actions/action"; 
+import { Link } from "react-router-dom";
+import { fetchCocktails } from "../actions/action"
+
+
+
+function CocktailList() {
+    const searchValue = useRef();
+    const { cocktails, loading } = useSelector((state) => ({ ...state.app }))
+    const [modifiedCocktail, setmodifiedCocktail] = useState([]);
+    const dispatch = useDispatch();
+
+
+    useEffect(() => {
+        dispatch(fetchCocktails());
+    }, []);
+
+    useEffect(() => {

```

Comment:
this mapping is not necessary, you can display cocktails without re-mapping property names